static/js/apps/biomed_nl/spinner.tsx

```diff
@@ -0,0 +1,97 @@
+/**
+ * Copyright 2025 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/** @jsxImportSource @emotion/react */
+
+/**
+ * A loading spinner that shows a new biomed pun every minute
+ */
+import { css, ThemeProvider } from "@emotion/react";
+import React, { ReactElement, useEffect, useRef, useState } from "react";
+
+import theme from "../../theme/theme";
+
+const THIRTY_SECONDS = 3000;
+const SIXTY_SECONDS = 60000;
+
+const customMessages = [
```

Comment:
nit: global constants should be UPPER_SNAKE_CASE

Reply:
fixed!